e2e/tests/api-driven/src/flowStatusHistory/flowStatusHistory.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Flow status history
+
+  @regression @flow-status-history
+  Scenario: Creating a flow
+    When a new flow is added
+    Then the status of the flow is online by default
+    And a flow_status_history record is created
```

Comment:
Not sure I agree with this assumption! 

I'd personally think that _existing_ flows should be "online by default", but newly added ones should be "offline by default" as that will match our user's mental models of "preparing to go live" ?

Reply:
This is a good shout - agree! Changes made in [8dc5368](https://github.com/theopensystemslab/planx-new/pull/3170/commits/8dc5368a9deb87e6e3735eabd01a46c714c09327)